### DISCUSSION TITLE
GPU OpenGL benchmark

### DISCUSSION THREAD

**Discussion by K1ngfish3r** [hardinfo2/hardinfo2#83] [General]
![image](https://github.com/user-attachments/assets/98d480fa-2982-4c21-87bd-ab0e57919b24)
Something ain't right when my 10 year old gpu beats out every other gpu on this list

**Comment by hwspeedy** [comment]
Yes, the OpenGL marks included are not so good - the OpenGL benchmark is new - so they are from development.

Please try the synchronize button - Then your GPU is beating even in this simple OpenGL test.

Lets gather some more benchmarks on OpenGL Benchmarking, then we can decide how to add/change features for OpenGL benchmarking.

You can watch the GPU benchmarks on project webpage (it's still WIP but works):
1. goto https://hardinfo2.org/
2. in the "Desktop" selector, choose GPU
3. goto Bench Stats and choose OpenGL

Please help the project by submitting your benchmarks by pressing the synchronize button in hardinfo2, thanx. 
PS: Program needs to be >=2.1.13 to have OpenGL benchmarks.
